# HG changeset patch
# User Noritada Kobayashi <nori1@dolphin.c.u-tokyo.ac.jp>
# Date 1220063096 -32400
# Node ID 1e07d3be4389d78b7b8404a29534ca045c1993b0
# Parent  b9aed5b27b3a336d0a69f6a9cf7341a32eb5eddc
Alert translators to usage of non-ASCII characters in confirmation messages. (Closes: #475802)

Since users (especially CJK users) should be able to input confirmation
messages without input methods, translations of those messages should
include nothing but ASCII characters.  This trial patch inserts comments
to alert translators to such cases.  Also, this patch changes a
confirmation message "Yes" and "No" to "YES" and "NO" respectively so
that they differ from menu dialog messages "Yes" and "No".

diff -r b9aed5b27b3a -r 1e07d3be4389 src/cmdline/cmdline_prompt.cc
--- a/src/cmdline/cmdline_prompt.cc	Mon Aug 25 16:10:25 2008 +0200
+++ b/src/cmdline/cmdline_prompt.cc	Sat Aug 30 11:24:56 2008 +0900
@@ -717,7 +717,10 @@
 	}
 
 
-      const string okstr=_("Yes"), abortstr=_("No");
+      // ForTranslators: This string is a confirmation message, which
+      // users (especially CJK users) should be able to input without
+      // input methods.  Please include nothing but ASCII characters.
+      const string okstr=_("YES"), abortstr=_("NO");
 
       while(1)
 	{
diff -r b9aed5b27b3a -r 1e07d3be4389 src/pkg_item.cc
--- a/src/pkg_item.cc	Mon Aug 25 16:10:25 2008 +0200
+++ b/src/pkg_item.cc	Sat Aug 30 11:24:56 2008 +0900
@@ -58,6 +58,9 @@
   using namespace widgets;
 }
 
+// ForTranslators: This string is a confirmation message, which users
+// (especially CJK users) should be able to input without input
+// methods.  Please include nothing but ASCII characters.
 static const char *confirm_str=N_("Yes, I am aware this is a very bad idea");
 
 static void try_delete_essential(wstring s,
